content/en/seachat/seachat-manual/08-incoming-queries/01-user-sends-an-image-message.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "User Sends an Image Message"
+description: "Learn how SeaChat understands and responds to user uploaded image messages."
+lead: ""
+date: 2025-01-22T08:48:45+00:00
+lastmod: 2025-01-22T08:48:45+00:00
+weight: 30
+draft: false
+images: []
+toc: true
+url: /en/seachat/manual/incoming-queries/user-sends-an-image-message
+---
+
+# Overview
+
+Up to this point, you've likely only seen SeaChat respond to text-based user chat messages. However, SeaChat has the capability to handle multimedia messages, such as audio and image, as well! This makes interactions more flexible and convenient for users, enabling them to share content in various formats for richer communication. 
+
+---
+
+## Image Understanding
+
+When a user uploads an image, SeaChat analyzes it to generate a detailed textual description. This description serves as the foundation for SeaChat's response, ensuring that the AI’s reply is tailored to the visual content. The process is designed to make it easier for users to share visual information without needing to describe or re-type it themselves.
+
+> :bulb: **Note** 
+> This feature is only available to premium subscribers. For non-premium accounts, SeaChat will only know that the user uploaded an image, but will not have any insight on the content.
+
+### Example
+
+In the following example, a user chats with the Seasalt.ai bot via WebChat and uploads a screenshot of a webinar invite in their message to the agent. By analyzing the contents of the image, SeaChat is able to understand and respond to the user's message effectively (without the user having to re-type all the contents of the image).
+
+<img width="60%" style="border-radius: 0.4rem" src="/images/seachat/en/incoming-queries/SeaChat_Responds_to_Image_Message_Webchat.png" alt="SeaChat is able to understand and respond to a user-uploaded image message in WebChat.">
```

Comment:
the image here looks too small and are not enlargeable. If you go to https://wiki.seasalt.ai/seachat/getting-started/02-quick-start/ and see the first image, you'll see a magnifying glass while hovering over the image and can click on it. Can you do the same with the 2 images in the PR?